src/includes/platforms/configuration/integrations/redux.mdx

```diff
@@ -0,0 +1,107 @@
+<PlatformContent includePath="configuration/redux-init" />
+
+If you have other enhancers or middleware such as `thunk`:
+
+```javascript
+const store = createStore(
+  rootReducer,
+  compose(applyMiddleware(thunk), sentryReduxEnhancer)
+);
+```
+
+<Alert level="info" title="Note">
+
+Sentry uses a redux **enhancer**. Pass the enhancer, as indicated above. Don't pass it to `applyMiddleware` and don't call the method when you pass it to the `createStore` method.
+
+</Alert>
+
+## Normalization Depth
+
+By default Sentry SDKs normalize any context to a depth of 3, which in the case of sending Redux state you probably will want to increase that. You do so by passing `normalizeDepth` to the `Sentry.init` call:
+
+```javascript
+Sentry.init({
+  dsn: "___DSN___",
+  normalizeDepth: 10, // Or however deep you want your state context to be.
+});
+```
+
+## Redux Enhancer Options
+
+Pass an options object as the first parameter to `Sentry.createReduxEnhancer` to configure it.
+
+<Alert level="warning" title="Note">
+
+We advise against sending sensitive information to Sentry, but if you do, we will try our best to filter out things such as user passwords.
+
+</Alert>
+
+`actionTransformer` (Function)
+
+: Used to remove sensitive information from actions. The first parameter passed to the function is the Redux action. Return `null` to not send the action to Sentry. By default we send all actions.
+
```

Comment:
```suggestion
`actionTransformer` (Function)

This can be used to remove sensitive information from actions. The first parameter passed to the function is the Redux action. We send all actions by default, if you don't want an action sent to Sentry, use `return null`.

```